tests/installation/partitioning_raid.pm

```diff
@@ -67,7 +67,7 @@ sub addpart {
         send_key_until_needlematch 'partition-selected-bios-boot-type', 'down';
     }
     else {
-        send_key_until_needlematch 'partition-selected-raid-type', 'down';
+        send_key_until_needlematch 'partition-selected-raid-type', 'down', 20, 3;
```

Comment:
Could you please a comment explaining why we use these args here? Then we can merge. Thanks!

Reply:
Done, could you please double check ? Let me know if you miss there anything. Ty!